core-blocks/embed/index.js

```diff
@@ -271,6 +271,7 @@ export const settings = getEmbedBlockSettings( {
 		from: [
 			{
 				type: 'raw',
+				priority: 0,
```

Comment:
Does priority need to be so low? While negatives are supported, I'd tend to think of zero as the lowest possible. I'm wondering if instead we should just _increase_ the priority value for paragraph, so that it's properly treated as a fallback if no others exist.

Reply:
>The problem with adding it to paragraph is that it's not for paragraph, it's for the embed block.

Adding it to paragraph is to reflect that paragraph is ultimately the fallback, and _anything_ else should take precedent over it, without needing to express its own priority. It may not be obvious to plugins that they'd need to assign priority, for example, and transforms might prefer paragraph when omitted.

>The priority on the paragraph was 20, which was also random? What's the default? 10? S this should be 9?

The numeric priorities are certainly imperfect 😄 `9` would reflect that we want it to take place before the baseline (if paragraph's considered the baseline), while still allowing others to be considered as more important. `0` is fine enough I guess, but leaves the impression that we'd never want anything else to take precedence over the embed's transform.

Reply:
Okay, makes sense. I'll change it.